src/actions/commands/actions.ts

```diff
@@ -2165,6 +2165,32 @@ class CommandEscInCommandline extends BaseCommand {
   }
 }
 
+@RegisterAction
+class CommandRemoveWordCommandline extends BaseCommand {
+  modes = [ModeName.CommandlineInProgress];
+  keys = ['<C-w>'];
+  runsOnceForEveryCursor() {
+    return this.keysPressed[0] === '\n';
```

Comment:
Is this here for a reason?

Reply:
runsOnceForEveryCursor()? I think it should be false since this action doesn't need to be execute twice when there are multiple cursors.

Reply:
Yeah I agree - let's just return `false`

Reply:
I think there are some other methods that need to change to return false as well, but I only corrected the methods I added